client/policy.go

```diff
@@ -18,6 +20,9 @@ func (self *ApiClient) Policy(projectId string) (Policy, error) {
 // PolicyUpdate updates a policy through the API
 func (self *ApiClient) PolicyUpdate(payload PolicyUpdatePayload) (Policy, error) {
 	var result Policy
+	if payload.ProjectId == "" {
+		return Policy{}, fmt.Errorf("projectId is required when updating policy")
+	}
```

Comment:
This should be validated by the resource schema, hence there's no use case where we'd be firing this API w/o project id